tool/testcase/template/ethereum.tmpl

```diff
@@ -1,6 +1,6 @@
 {{ define "ethereum" }}
 {
-    Network: network.Ethereum,
+    Network: network.{{ UppercaseFirst .Network }},
```

Comment:
Maybe adding "network.Ethereum" as the default network is needed?  if UppercaseFirst is null then this may lead to unwanted behavior?